docs/sidebars.js

```diff
@@ -149,6 +154,6 @@ module.exports = {
       href: 'https://status.flashbots.net/',
       label: 'Status',
     },
-        'policies/privacy','policies/terms-of-service',
```

Comment:
Are these links? If so I think lowercase is actually what we want here since the files have lowercase names, but it might be case-insensitive, in which case it doesn't matter.

Reply:
I changed the file names too in an attempt to fix the caps - it was unnecessary but harmless

Reply:
the tl;dr is everything renders on 3000

Reply:
fixed for good measure